system/liquid-ceph/templates/deployment.yaml

```diff
@@ -70,21 +71,13 @@ spec:
             value: {{ quote $.Values.global.region }}
           - name: LIQUID_POLICY_PATH
             value: "/etc/liquid/policy.yaml"
-          - name: CEPH_ACCESS_KEY
-            valueFrom:
-              secretKeyRef:
-                name: {{ .Values.ceph.rgwSecretName }}
-                key: AccessKey
-          - name: CEPH_SECRET_KEY
-            valueFrom:
-              secretKeyRef:
-                name: {{ .Values.ceph.rgwSecretName }}
-                key: SecretKey
           securityContext:
             runAsNonRoot: true
           volumeMounts:
           - mountPath: /etc/liquid
             name: policy
+          - mountPath: /etc/liquid
+            name: zone-mapping
```

Comment:
I think the mount points collide. I would propose to move `zone-mappings.yaml` and `policies.yaml` in a single ConfigMap (e.g. called just `liquid-ceph`). Then mounting that single ConfigMap would provide both files.